.ci/pytorch/common_utils.sh

```diff
@@ -56,7 +56,25 @@ function assert_git_not_dirty() {
 function pip_install_whl() {
   # This is used to install PyTorch and other build artifacts wheel locally
   # without using any network connection
-  python3 -mpip install --no-index --no-deps "$@"
```

Comment:
@ethanwee1 Sanity check: wouldn't removing the flags from this line be the simpler change?